### DISCUSSION TITLE
Holograms

### DISCUSSION THREAD

**Discussion by Skizzles** [BG-Software-LLC/WildLoaders#12] [Pending Feature Requests]
Add holograms per loader that can display information.

**Comment by harunn416** [comment]
I'm also looking for this feature! It would be great if we could display placeholders like {5} for the loader type or radius in the hologram-lines. This would help players distinguish between different loader ranges visually. Is there any update on this?

**Reply by OmerBenGera** [reply]
Added in latest dev build :)
